common/presenters/population-table/locations-to-population-table-component.js

```diff
@@ -0,0 +1,70 @@
+const { parseISO, isValid } = require('date-fns')
+
+const tablePresenters = require('../table')
+
+const { dayConfig, establishmentConfig } = require('./population-table-config')
+
+function locationsToPopulationTable({
+  isSortable = false,
+  query,
+  focusDate,
+  startDate,
+} = {}) {
+  if (!isValid(startDate)) {
+    startDate = parseISO(startDate)
+  }
+
+  if (!isValid(startDate)) {
+    startDate = new Date()
+  }
+
+  const tableConfig = [
+    {
+      ...establishmentConfig({}),
+    },
+    {
```

Comment:
Could this potentially use some kind of loop? The only difference I can see for all of these blocks is the `baseOffset` increasing by one which could use an index?

Reply:
Replaced this with a loop